src/main/java/com/selenium/course/pages/ChatterPage.java

```diff
@@ -110,31 +119,58 @@ public WebElement getOptionDeleteFromLatestPost() {
         return null;
     }
 
-    public void removeLatestPost(){
-        WebElement deleteLatestPost = getOptionDeleteFromLatestPost();
+    public ChatterPage removeLatestPost(){
+        WebElement deleteLatestPost = getDeleteElementFromLatestPost();
         deleteLatestPost.click();
+        Alert alert = driver.switchTo().alert();
+        alert.accept();
+        return this;
     }
 
     public void expandLatestDropDown(){
         By by  = By.xpath("//span[@class='feeditemActionMenu']//a[contains(@id,'spillovermenu')]");
         List<WebElement> dropBoxMenu = driver.findElements(by);
```

Comment:
This variable of method should be an attribute of class using Page Factory even if it is a list of webelements
for example
@FindBy(xpath = "//span[@class='feeditemActionMenu']//a[contains(@id,'spillovermenu')]")
 List<WebElement> dropBoxMenu;